ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_7.yaml

```diff
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Enable an Accelerator Profile From The Grid of the Accelerator Profile View"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  generation: 1
+  name: qeacceleratorprofile7
+  namespace: redhat-ods-applications
+spec:
+  displayName: qe_accelerator_profile_7
+  enabled: false
```

Comment:
Shouldn't it be "true" (enable) ?

Reply:
In the test I'm enabling the AP, I need to provision one disabled.